app/src/test/java/nerd/tuxmobil/fahrplan/congress/sharing/JsonLectureFormatTest.kt

```diff
@@ -0,0 +1,35 @@
+package nerd.tuxmobil.fahrplan.congress.sharing
+
+import com.google.common.truth.Truth.assertThat
```

Comment:
Your test fails on CI because `testImplementation Libs.truth` is missing in _app/build.gradle_.

Please note that I updated the checklist a few days before.

Reply:
Fixed in 413ed4b6bee4c442d87e5fe87ca228abbe1b98f0.